app/javascript/components/admin/manage_users/ManageUserRow.jsx

```diff
@@ -21,6 +21,10 @@ export default function ManageUserRow({ user }) {
   const { t } = useTranslation();
   const mutationWrapper = (args) => useCreateServerRoom({ userId: user.id, ...args });
   const updateUserStatus = useUpdateUserStatus();
+  const event = new Date(user?.created_at);
+  const options = {
+    year: 'numeric', month: 'long', day: 'numeric', hour: 'numeric', minute: 'numeric',
+  };
```

Comment:
This is being duplicated in every file. If we want to change the format, we have to change it in a ton of places. Can we find a smarter way to do this? (ie helpers?)